libs/common/src/lib/ai-models/answer-generation/answer-generation.component.html

```diff
@@ -54,13 +63,19 @@
                         resizable
                         [formControlName]="field.key"
                         [rows]="2"
-                        [help]="field.value.info">
+                        [help]="
+                          (userKeys.required.includes(field.key) ? (required | async) + ' ' : '') +
+                          (field.value.info || '')
+                        ">
                         {{ field.value.title }}
                       </pa-textarea>
                     } @else {
                       <pa-input
                         [formControlName]="field.key"
-                        [help]="field.value.info">
+                        [help]="
+                          (userKeys.required.includes(field.key) ? (required | async) + ' ' : '') +
+                          (field.value.info || '')
```

Comment:
if there is no `info`, we will end up with an empty `()`

Reply:
I just realised my mistake, those `()` are for the expression, not a string. So forgot about this comment

Reply:
I just tested it. It seems fine, the parentheses are not rendered